components/how-to.tsx

```diff
@@ -0,0 +1,80 @@
+export default function HowTo() {
+  return (
+    <div className="howto md:pl-16 md:pr-16 lg:p-10" id="howto">
+      <span>
+        <p className="text-2xl text-bold howto-header text-shadow">
+          Step 1 - Wrap Your Gift → Give
+        </p>
+        <p className="text-2xl font-extralight	">
+          To mint your own Raging Santa you must provide a gift NFT of your
+          choice for the Raging Santas Gift Pool.
+          <br />
+          <br />
+          Each Raging Santa costs 0.03 ETH + gas to mint.
+          <br />
+          <br />
+          Your Raging Santa will be proof that you gave generously to the Gift
+          Pool and will also be proof of your claim to a Free Gift on X-Mas Day!
```

Comment:
I think we should be consistent on "X-Mas" or "Christmas" 🤔 maybe "Christmas"?

Reply:
good call - i like XMas bc its shorter and it has an X. but down for whatever